mteb/tasks/Clustering/eng/ArxivClusteringS2S.py

```diff
@@ -38,6 +38,14 @@ class ArxivClusteringS2S(AbsTaskClustering):
 }""",
         descriptive_stats={
             "n_samples": {"test": 732723},
-            "avg_character_length": {"test": 74},
+            "avg_character_length": {
+                "test": {
+                    "average_label_count": 23636.225806451614,
+                    "average_text_length": 23636.225806451614,
```

Comment:
This seems like a bug?

Reply:
It's not working correctly, because it contains a list of lists for sentences and labels. I will remove metadata update